tests/unit/fs_events_test.ts

```diff
@@ -32,7 +32,7 @@ async function getTwoEvents(
   const events = [];
   for await (const event of iter) {
     events.push(event);
-    if (events.length > 2) break;
+    if (events.length >= 2) break;
```

Comment:
Funny that this function name is `getTwoEvents` and waits for 3 instead. Spotted this bug by switching to the kqueue backend